docs/source/models/causes/other_causes/index.rst

```diff
@@ -74,16 +74,21 @@ States Data
      - 
    * - C
      - disabilty weight
-     - all_cause_yld_rate - anemia_yld_rate - population_maternal_disorders_yld_rate + pregnancy_maternal_disorders_yld_rate
+     - all_cause_yld_rate - anemia_yld_rate - population_maternal_disorders_yld_rate 
      - See extended note below
 
 .. note::
 
   Note that we are assuming the YLD rate due to all causes other than maternal disorders (including YLDs due to anemia) do not significantly vary between the pregnant/postpartum not pregnant/postpartum populations. We do rescale the rate of maternal disorders from the general population of women of reproductive age to the rate among the pregnant/postpartum population in this equation. 
 
-  Note that we include YLDs due to maternal disorders in the disability weight of "background morbidity" here despite the fact that maternal disorders is included as a cause of disability within the nutrition optimization pregnancy simulation. This is because in the simulation, simulants acquire pre-COMO-adjusted maternal disorders YLDs over the course of a single timestep in which accumulation of YLDs due to all other causes besides maternal disorders is paused. Therefore, since maternal disorders YLDs are not accumulated during the remainder of the simulation when anemia YLDs are accumulated, anemia YLDs would not be adjusted for comorbidity with maternal disorders unless they were included in the disability weight for background morbidity, which will be present during the timesteps for which anemia YLDs are accumulated.
+  Note that we are excluding YLDs due to maternal disorders in YLDs due to background morbidity, despite that YLDs due to maternal disorders in the nutrition optimization model are modeled separately and are never comorbid with other conditions. This limits us in that we therefore do not capture:
 
-    Note that while this allows us to more accurately estimate YLDs averted between scenarios, it will cause double counting of YLDs due to maternal disorders in the total amount of YLDs accrued within the simulation (due to all modeled causes combined).
+    1. any increase in maternal disorders YLDs due to reductions in anemia YLDs through the COMO adjustment,
```

Comment:
When you say "COMO adjustment" here, you are including the comorbidity calculations in Vivarium, I think? In my head "COMO" specifically meant the GBD process, so this took me a second.

Reply:
Yeah, I have been using "COMO" loosely here as basically our application of the GBD COMO process. I'll add a note that indicates that.